test/e2e/network_segmentation.go

```diff
@@ -697,8 +697,8 @@ var _ = Describe("Network Segmentation", func() {
 				DescribeTable(
 					"can be accessed to from the pods running in the Kubernetes cluster",
 					func(netConfigParams networkAttachmentConfigParams, clientPodConfig podConfiguration) {
-						if isLocalGWModeEnabled() {
-							const upstreamIssue = "https://github.com/ovn-org/ovn-kubernetes/pull/4554"
+						if netConfigParams.topology == "layer2" && isLocalGWModeEnabled() {
```

Comment:
IMO it's better to use `IsGatewayModeLocal()` instead.  It reads the annotation on the control plane node instead of relying on env vars.  Simplifies local testing.

Reply:
ack sounds like I added this util huh: https://github.com/ovn-org/ovn-kubernetes/commit/044ead332e4e26cbc141afc0739b37b68bdb76c7#diff-65b73a6a62a4e5b8903b9f17f0e03a42365b127e7d86f006f9bec363b4be04b8R405

don't even remember adding it...